src/Microsoft.DotNet.Darc/src/DarcLib/Actions/Remote.cs

```diff
@@ -739,12 +742,16 @@ private IEnumerable<DependencyDetail> CalculateLeavesOfCoherencyTrees(IEnumerabl
             List<DependencyDetail> itemsToUpdate,
             string message)
         {
+            IEnumerable<DependencyDetail> oldDependencies = await GetDependenciesAsync(repoUri, branch, loadAssetLocations: true);
+            itemsToUpdate = (await AddAssetLocationToDependenciesAsync(itemsToUpdate)).ToList();
```

Comment:
>AddAssetLocationToDependenciesAsync [](start = 35, length = 35)

You can make this void. No need to re-assign it to itemsToUpdate since itemsToUpdate will be modified inside the method and retain the value. Also you'd be avoiding to convert it to a list

Reply:
It's just a pattern I'm used to following. Return the thing you modified even if the parameter got changed. I can make it void and turn that into an out param.

Reply:
discussed offline. Leaving as is.